exonum-java-binding-qa-service/src/test/java/com/exonum/binding/qaservice/transactions/QaTransactionConverterTest.java

```diff
@@ -51,76 +44,31 @@ void hasFactoriesForEachTransaction(QaTransaction tx) {
     short id = tx.id();
 
     assertThat(QaTransactionConverter.TRANSACTION_FACTORIES)
-            .as("No entry for transaction %s with id=%d", tx, id)
-            .containsKey(id);
+        .as("No entry for transaction %s with id=%d", tx, id)
+        .containsKey(id);
   }
 
   @Test
   void toTransactionTransactionOfAnotherService() {
-    BinaryMessage message = new Message.Builder()
-        .setServiceId((short) (QaService.ID + 1))
-        .setMessageType(QaTransaction.INCREMENT_COUNTER.id())
-        .setBody(ByteBuffer.allocate(0))
-        .setSignature(new byte[Message.SIGNATURE_SIZE])
-        .buildRaw();
+    RawTransaction tx = RawTransaction.newBuilder()
+        .serviceId((short) (QaService.ID + 1))
+        .transactionId(QaTransaction.INCREMENT_COUNTER.id())
+        .payload(ByteBuffer.allocate(0).array())
+        .build();
 
     Exception e = assertThrows(IllegalArgumentException.class,
-        () -> converter.toTransaction(message));
+        () -> converter.toTransaction(tx));
     assertThat(e).hasMessageMatching("Wrong service id \\(\\d+\\), must be "
         + QaService.ID);
   }
 
   @Test
   void toTransactionUnknownTransaction() {
-    UnknownTx unknownTx = new UnknownTx();
-    BinaryMessage message = unknownTx.getMessage();
+    RawTransaction message = UnknownTx.createRawTransaction();
 
     Exception e = assertThrows(IllegalArgumentException.class,
         () -> converter.toTransaction(message));
     assertThat(e).hasMessageStartingWith("Unknown transaction");
   }
 
-  @ParameterizedTest
-  @MethodSource("transactionMessages")
-  void toTransaction(Class<? extends Transaction> expectedType, Message messageTemplate) {
```

Comment:
Why this test has been removed? It tests the _main responsibility_ of this class — converting RawTransactions to Transactions.